Islandora-Devops/isle-dc
# All wired up

Depends on https://github.com/Islandora-Devops/isle-alpaca/pull/3

Comment in the build sections for and point to ^^ once it's merged (or pull it in and point to a local copy on disk).

## How to Test

From your isle-dc folder:
- `docker-compose down -v`
- `rm -rf codebase && mkdir codebase` to clean the codebase
- If you haven't already done so, `(cd scripts; ./generate_keys.sh)` to generate keys
- `docker-compose up --build -d`
-  `docker exec -it isle_dc_proto_solr bash -c "solr create_core -c islandora"`
- `docker exec -it isle_dc_proto_php bash -c "sh /scripts/islandora/install-islandora.sh"`

You can test by ingesting a repository item, giving it a media and tagging that media as an original file.  This should trigger derivatives.

Admittedly, there's still something super wonky with karaf/camel uploading derivatives, and sometimes you get uploaded files of 0.  I'm still working through that, but you can at least see that the whole system is functioning, albeit a bit impaired.
